superset/utils/pandas_postprocessing/histogram.py

```diff
@@ -53,7 +53,7 @@ def histogram(
         raise ValueError(f"The column '{column}' must be numeric.")
 
     # calculate the histogram bin edges
-    bin_edges = np.histogram_bin_edges(df[column], bins=bins)
+    bin_edges = np.histogram_bin_edges(df[column].dropna(), bins=bins)
```

Comment:
I added the `dropna()` to deal with `NULL` values independently of the query results.